.github/workflows/test-daily.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: 'Run daily tests'
+
+on:
+  push:
+  schedule:
+    - cron: '14 5 * * *'
+  workflow_call:
+    inputs:
+      branch-name:
+        description: 'Checkout to a specific branch'
+        required: true
+        default: 'master'
+        type: string
+
+jobs:
+  run-rec-bulk-test:
+    if: ${{ vars.REC_BULKTEST_USE_UBICLOUD == '1' }}
```

Comment:
this is a bit misnamed. if it is set to 0, the test does not run at all, instead of running on "not ubicloud"

Reply:
I'll rename and move the code to misc-dailies, although we run the risk this becoming a hotchpot of unrelated workflows that only have in common that they are run daily.

Reply:
Ah, no, I just meant the var name. As for moving, I'm torn between "we have too many workflows" and "workflows are a good tool for organising things in a way that makes sense". Thinking about it, I realise another important feature of workflows is that they can be disabled/enabled from the UI. So keeping things a bit separate makes sense.

Reply:
UBICLOUD_DAILY_REC_BULKTEST?